pydantic_evals/pydantic_evals/evaluators/evaluator.py

```diff
@@ -99,6 +100,15 @@ def downcast(self, *value_types: type[T]) -> EvaluationResult[T] | None:
         return None
 
 
+@dataclass
+class EvaluatorFailure:
+    """Represents a failure raised during the execution of an evaluator."""
+
+    name: str
+    error_msg: str
```

Comment:
I don't love unnecessary abbreviations in field names, why not just `error_message`?